src/index.html

```diff
@@ -1,17 +1,134 @@
+        </label>
+        <label class="field"> Name: <input type="text" name="name"> </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="years"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field">
+          Full date of birth: <input type="date" name="birthDate">
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input type="checkbox" name="term">
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required=""
+            name="email"
+            maxlength="20"
+          >
+        </label>
+        <label class="field">
+          Password:
+          <input
+           type="password"
+           name="password"
+           minlength="6"
+           required=""
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="cat"
+              value="yes"
+            > Yes
+          </label>
+          <label> 
+            <input
+              type="radio"
+              name="cat"
+              value="no"
+            > No 
+          </label>
+        </div>
+        <label class="field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000000"
+          >
+        </label>
+        <label class="field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeToBed"
+            step="2"
+          >
+        </label>
+        <label class="field">
+          What are your favorite brand of cars?
+          <select name="brandOfCars" multiple="">
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="mark"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <label class="field">
+          Comments: <textarea name="Comments" autocomplete="off"> </textarea>
+        </label>
+        <label class="field">
+          Would you recommend us?
+          <select name="recommendation">
+            <option value="yes">yes</option>
+          </select>
```

Comment:
as I think we need "no" here
<img width="291" alt="image" src="https://user-images.githubusercontent.com/86183007/214787525-e84e6394-d3ba-4d98-ba62-397188ca153c.png">